code/modules/mob/living/carbon/carbon.dm

```diff
@@ -86,6 +86,8 @@
 		"<span class='userdanger'>You feel a powerful shock coursing through your body!</span>", \
 		"<span class='danger'>You hear a heavy electrical crack.</span>" \
 	)
+	if(prob(25) && heart_attack)
+		heart_attack = 0

```

Comment:
This is a really brilliant feature, being able to electrocute yourself if you have a heart attack. The sort of little detail you rarely see.